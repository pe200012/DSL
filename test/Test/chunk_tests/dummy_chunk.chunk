Chunk
    { _header = Header
        { _signature = 457995617
        , _version = 84
        , _format = 0
        , _luacData = "\EM\147\r\n\SUB\n"
        , _instructionSize = 4
        , _luaIntegerSize = 8
        , _luaNumberSize = 8
        , _luacInt = 22136
        , _luacNum = 370.5
        }
    , _function = Function
        { _srcFile = ""
        , _lineDefined = 0
        , _lastLineDefined = 0
        , _numParams = 0
        , _isVararg = 1
        , _maxStackSize = 2
        , _code = []
        , _constants = []
        , _upvalues = []
        , _protos = []
        , _debug = DebugInfo
            { _debugLineInfo = ""
            , _debugAbsLineInfo = []
            , _debugLocals = []
            , _debugUpvalues = []
            }
        }
    }