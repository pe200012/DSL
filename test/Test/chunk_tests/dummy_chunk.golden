
main <test/Test/chunk_tests/dummy_chunk.dump:0,0> (2 instructions at 0x15b607370)
0+ params, 2 slots, 1 upvalue, 0 locals, 0 constants, 0 functions
	1	[1]	VARARGPREP	0
	2	[1]	RETURN   	0 1 1	; 0 out
constants (0) for 0x15b607370:
locals (0) for 0x15b607370:
upvalues (1) for 0x15b607370:
	0	_ENV	1	0
